### PR TITLE
test(#t04): search job by title

### PR DESCRIPTION
**Pull request by tminhbao**

### CODE REVIEW

File: src/components/Pagination/Pagination.test.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        />
+      </CustomProvider>
+    );
+    const paginationListItem = await findAllByTestId("pagination-item");
+    const paginationItemActice = await findByTestId("pagination-item active");
+    expect(paginationListItem).toHaveLength(9);
+    expect(paginationItemActice).not.toBeNull();
+  });
+
+  //   it("should render 10 jobs per page", async () => {
```
Comment (TruongNP) [inline src/components/Pagination/Pagination.test.tsx:40]: Please remove if not use

File: src/components/Pagination/Pagination.test.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          currentPage={1}
+          handleChangeCurrentPage={changeCurrentPage}
+        />
+      </CustomProvider>
+    );
+    const paginationItemActice = await findByTestId("pagination-item active");
+    expect(paginationItemActice.textContent).toEqual("1");
+  });
+
+  //   it("should active correctly page when clicked", async () => {
```
Comment (TruongNP) [inline src/components/Pagination/Pagination.test.tsx:89]: Please remove if not use

File: src/components/Pagination/Pagination.test.tsx (hunk 1 above)
Comment (tminhbao) [reply src/components/Pagination/Pagination.test.tsx:40]: Fixed

File: src/components/Pagination/Pagination.test.tsx (hunk 2 above)
Comment (tminhbao) [reply src/components/Pagination/Pagination.test.tsx:89]: Fixed

**Review by TruongNP — APPROVED** [review]

**Pull request merged**